src/content/Angular-CSharp/ClientApp/src/app/nav-menu/nav-menu.component.html

```diff
@@ -37,10 +37,12 @@
               >Fetch data</a
             >
           </li>
+          <!--#if (IndividualLocalAuth) -->
+          <li class="nav-item">
+              <a class="nav-link text-dark" href="/Identity/Account/Manage">Account</a>
```

Comment:
I include a general "Account" link to the razor pages. It's sad that that we no longer statefully indicate whether you're logged in already and show your logged in username in the nav bar. This should be made possible with our samples that use `MapIdentityApi<TUser>()`.

Unlike the razor pages nav bar, we don't specify the `asp-area` and `asp-page` tag helpers for link generation, but this does not seem any worse than the `applicationPaths.IdentityRegisterPath` in `api-authorization.constants.ts` which also hardcoded `/Identity/Account` but is now deleted. We could try rendering server-produced links that we fetch somehow, but you ultimately need the source of truth somewhere.